### PR TITLE
Feature/leaderboard UI

### PR DESCRIPTION
**Pull request by raineaeternal**
Finished the UI. Next is to make

### CODE REVIEW

File: AccSaber/Installers/MenuInstaller.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        private readonly SiraLog _log;
+        public MenuInstaller(SiraLog log)
+        {
+            _log = log;
+        }
+        
+        public override void InstallBindings()
+        {
+            Container.BindInterfacesAndSelfTo<AccSaberLeaderboardViewController>().FromNewComponentAsViewController().AsSingle().NonLazy();
+            Container.BindInterfacesAndSelfTo<AccSaberPanelController>().FromNewComponentAsViewController().AsSingle().NonLazy();
```
Comment (PulseLane) [inline AccSaber/Installers/MenuInstaller.cs]: Can lazily bind the panel and leaderboard view controllers

File: AccSaber/UI/Leaderboard/LeaderboardViewController.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using LeaderboardCore.Models.UI.ViewControllers;
+using SiraUtil.Tools;
+using Zenject;
+
+
+namespace AccSaber.UI.Leaderboard
+{
+    public class AccSaberLeaderboardViewController : BasicLeaderboardViewController, INotifyPropertyChanged
```
Comment (PulseLane) [inline AccSaber/UI/Leaderboard/LeaderboardViewController.cs]: no reason to be using INotifyPropertyChanged

File: AccSaber/UI/Leaderboard/LeaderboardViewController.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        [Inject] private SiraLog _log;
```
Comment (PulseLane) [inline AccSaber/UI/Leaderboard/LeaderboardViewController.cs]: np: put attribute on line above

File: AccSaber/UI/Leaderboard/LeaderboardUI.bsml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        </vertical>
+        <vertical active='~is-loading' pref-width='80' pref-height='10' >
+            <vertical pref-width='10'>
+                <loading-indicator preserve-aspect='true'/>
+            </vertical>
+        </vertical>
+    </horizontal>
+    <page-button direction='Up' anchor-pos-x='-37.75' anchor-pos-y='-7' active='~up-enabled' pref-width='5' on-click='up-clicked' />
+    <page-button direction='Down' anchor-pos-x='-37.75' anchor-pos-y='-34' active='~down-enabled' pref-width='5' on-click='down-clicked' />
+</bg>
```
Comment (PulseLane) [inline AccSaber/UI/Leaderboard/LeaderboardUI.bsml]: unused file

File: AccSaber/Models/AccTypesList.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        public enum AccTypes
+        {
+            Overall,
+            True,
+            Standard,
+            Tech
+        }
+    }
+}
```
Comment (PulseLane) [inline AccSaber/Models/AccTypesList.cs]: unused enum/class

File: AccSaber/manifest.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/bsmg/BSIPA-MetadataFileSchema/master/Schema.json",
+  "id": "AccSaber",
+  "name": "AccSaber",
+  "author": "Raine",
```
Comment (PulseLane) [inline AccSaber/manifest.json]: AccSaber Team

File: AccSaber/manifest.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/bsmg/BSIPA-MetadataFileSchema/master/Schema.json",
+  "id": "AccSaber",
+  "name": "AccSaber",
+  "author": "Raine",
+  "version": "1.0.0",
```
Comment (PulseLane) [inline AccSaber/manifest.json]: keep as <1.0 until ready for release

File: AccSaber/manifest.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
+﻿{
+  "$schema": "https://raw.githubusercontent.com/bsmg/BSIPA-MetadataFileSchema/master/Schema.json",
+  "id": "AccSaber",
+  "name": "AccSaber",
+  "author": "Raine",
+  "version": "1.0.0",
+  "description": "Plugin for AccSaber in-game leaderboard.",
+  "gameVersion": "1.17.0",
+  "dependsOn": {
```
Comment (PulseLane) [inline AccSaber/manifest.json:9]: missing LeaderboardCore

File: AccSaber.user [hunk 9]
```diff
@@ -0,0 +1,14 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (PulseLane) [inline AccSaber.user]: *.user file - should be gitignored

File: AccSaber.sln.DotSettings.user [hunk 10]
```diff
@@ -0,0 +1,9 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (PulseLane) [inline AccSaber.sln.DotSettings.user]: *.user file - should be gitignored

File: AccSaber/Installers/MenuInstaller.cs [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+using AccSaber.UI.Panel;
+using SiraUtil;
+using SiraUtil.Tools;
+using Zenject;
+
+namespace AccSaber.Installers
+{
+    public class MenuInstaller : Installer
+    {
+        private readonly SiraLog _log;
```
Comment (PulseLane) [inline AccSaber/Installers/MenuInstaller.cs]: np: no reason to store log if you're not using it.

File: AccSaber/Managers/AccSaberManager.cs [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        private CustomLeaderboardManager _customLeaderboardManager;
+        
+        private readonly ViewController _accSaberPanelController;
+        protected override ViewController panelViewController => _accSaberPanelController;
+        
+        private readonly ViewController _mainLeaderboardViewController;
+        protected override ViewController leaderboardViewController => _mainLeaderboardViewController;
+        private SiraLog _log;
+        public AccSaberManager(AccSaberPanelController accSaberPanelController, AccSaberLeaderboardViewController mainLeaderboardViewController, CustomLeaderboardManager customLeaderboardManager, SiraLog log)
```
Comment (PulseLane) [inline AccSaber/Managers/AccSaberManager.cs:22]: np: newline before ctor

File: AccSaber/Plugin.cs [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+namespace AccSaber
+{
+	[Plugin(RuntimeOptions.DynamicInit)]
+	public class Plugin
+	{
+		[Init]
+		public void Init(Logger logger, Config config, Zenjector zenjector)
+		{
+			zenjector.On<PCAppInit>().Pseudo(Container => Container.BindLoggerAsSiraLogger(logger));
+			zenjector.OnMenu<MenuInstaller>().ShortCircuitForMultiplayer();
```
Comment (PulseLane) [inline AccSaber/Plugin.cs]: ShortCircuitForMultiplayer not necessary for a menuinstaller

File: AccSaber/Plugin.cs [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+		}
+
+		[OnEnable, OnDisable]
+		public void OnStateChanged()
```
Comment (PulseLane) [inline AccSaber/Plugin.cs]: empty function, not necessary

File: README.md [hunk 15]
```diff
@@ -1,4 +1,4 @@
-# Accaber-Plugin
+# AccSaber-Plugin
```
Comment (PulseLane) [inline README.md:1]: Since you're changing the readme anyway - fix the "ScoreRequirement" name in "Contributing" section. Also, oxford comma :)

**Review by PulseLane — CHANGES_REQUESTED** [review]

**Review by raineaeternal — COMMENTED** [review]
Changed files with changes you wanted.

File: AccSaber/Installers/MenuInstaller.cs [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+using AccSaber.UI.Panel;
+using SiraUtil;
+using SiraUtil.Tools;
+using Zenject;
+
+namespace AccSaber.Installers
+{
+    public class MenuInstaller : Installer
+    {
+        public MenuInstaller()
```
Comment (PulseLane) [inline AccSaber/Installers/MenuInstaller.cs]: unnecessary empty ctor

Comment (raineaeternal) [reply AccSaber/Installers/MenuInstaller.cs]: Removed in commit e67b45a

**Comment by raineaeternal** [comment]
Removed user files from repo

**Review by PulseLane — APPROVED** [review]

**Pull request merged**